lua/cmp/utils/str.lua

```diff
@@ -153,4 +153,10 @@ str.escape = function(text, chars)
   return table.concat(escaped, '')
 end
 
+--- @param c string|nil the string to check
+--- @return boolean is_whitespace `true` if `s` matches `%s*`
+str.is_whitespace_char = function(c)
+  return c == '' or c == ' '
+end
```

Comment:
It's OK for now, but I think I'll fix it later because '' isn't whitespace.

Reply:
Maybe `is_visible` would be better.